[2024-01-05T16:08:49.307+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_metadata_collector.send_metadata_to_kafka scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-05T16:08:49.312+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_metadata_collector.send_metadata_to_kafka scheduled__2024-01-01T00:00:00+00:00 [queued]>
[2024-01-05T16:08:49.312+0000] {taskinstance.py:2171} INFO - Starting attempt 16 of 16
[2024-01-05T16:08:49.318+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): send_metadata_to_kafka> on 2024-01-01 00:00:00+00:00
[2024-01-05T16:08:49.321+0000] {standard_task_runner.py:60} INFO - Started process 146 to run task
[2024-01-05T16:08:49.324+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_metadata_collector', 'send_metadata_to_kafka', 'scheduled__2024-01-01T00:00:00+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/dag_metadata_collector.py', '--cfg-path', '/tmp/tmpjsilv1x8']
[2024-01-05T16:08:49.325+0000] {standard_task_runner.py:88} INFO - Job 42: Subtask send_metadata_to_kafka
[2024-01-05T16:08:49.352+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_metadata_collector.send_metadata_to_kafka scheduled__2024-01-01T00:00:00+00:00 [running]> on host 73232b21783f
[2024-01-05T16:08:49.389+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_metadata_collector' AIRFLOW_CTX_TASK_ID='send_metadata_to_kafka' AIRFLOW_CTX_EXECUTION_DATE='2024-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='16' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-01T00:00:00+00:00'
[2024-01-05T16:12:46.504+0000] {local_task_job_runner.py:302} WARNING - State of this instance has been externally set to success. Terminating instance.
[2024-01-05T16:12:46.507+0000] {process_utils.py:131} INFO - Sending 15 to group 146. PIDs of all processes in the group: [146]
[2024-01-05T16:12:46.508+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 146
[2024-01-05T16:13:46.515+0000] {process_utils.py:149} WARNING - process psutil.Process(pid=146, name='airflow task ru', status='sleeping', started='16:08:48') did not respond to SIGTERM. Trying SIGKILL
[2024-01-05T16:13:46.518+0000] {process_utils.py:86} INFO - Sending the signal 9 to group 146
[2024-01-05T16:13:46.526+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=146, name='airflow task ru', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='16:08:48') (146) terminated with exit code -9
[2024-01-05T16:13:46.527+0000] {standard_task_runner.py:175} ERROR - Job 42 was killed before it finished (likely due to running out of memory)
